### DISCUSSION TITLE
How to change border top radius of chat bubbles

### DISCUSSION THREAD

**Discussion by mogulist** [GetStream/stream-chat-react-native#614] [Q&A]
I was able to change border bottom radius of chat bubbles by setting `borderRadiusL` and `borderRadiusS` in  `messageSimple.container`:
```
export const theme: DeepPartial<Theme> = {
    messageSimple: {
        content: {
            container: {
                borderRadiusL: 6,
                borderRadiusS: 6,
            },
           . . .
};
```

As shown below, border top radius are not changed:
![image](https://user-images.githubusercontent.com/25053780/115166608-f94a7880-a0ee-11eb-97ff-4bec7d692457.png)

I couldn't find any property to change border top radius of chat bubbles.

**Comment by treyshedrick** [comment]
Following up on this. Were you able to figure this out? @mogulist
